docs/recipes/babelrc.md

```diff
@@ -30,6 +30,21 @@ You can override the default Babel configuration AVA uses for test transpilation
 }
 ```
 
+## Provide ES2015 Polyfills to your tests
```

Comment:
Perhaps `Polyfill tests`? I'm trying to avoid `ES2015` mentions since the standard is continuously evolving. AVA tries to support stage-4 syntax which means currently we're supporting more than ES2016, but not ES2017 since it isn't official yet.